raytracing/imagingpath.py

```diff
@@ -310,6 +310,13 @@ def displayRange(self):
             for (planePosition, magnification) in conjugates:
                 if displayRange < self.objectHeight * magnification:
                     displayRange = self.objectHeight * magnification
+
+        if axes is not None:
+            if len(axes.lines) != 0:
```

Comment:
Ouch: a bit of a hack. We sometimes use lines for other things, this is not particularly safe or robust.

Reply:
Indicate FIXME: in a comment.